### ISSUE TITLE
Migrating Js to React js 

### ISSUE THREAD

**Issue by Akshat111111** [Akshat111111/new-and-easy-Travel#2]
Write the application in reactjs

**Comment by k0msenapati** [comment]
If we are migrating to next js we don't need react js

**Comment by Akshat111111** [comment]
> If we are migrating to next js we don't need react js

yeah, but inorder to demonstrate practically the features of nextjs as compared to react js, its important

**Comment by TomLucasakaTGeek** [comment]
@Akshat111111 sir I can migrate vanilla js to reactjs

**Comment by Akshat111111** [comment]
any progress??

**Comment by TomLucasakaTGeek** [comment]
Yes sir on it, but it will take some time as converting complete javascript to React is .... yk.

**Comment by TomLucasakaTGeek** [comment]
Sir I am adding the React in a new folder called **"frontend"** as of right now, if this is ok then its fine but if you want to keep the name of the folder as **"views"** I will _rename_ it once the migration is completed from _**ejs -> jsx**_.

**Comment by Akshat111111** [comment]
The frontend is fine.Dont forget to add its deployment link on the readme file

**Comment by TomLucasakaTGeek** [comment]
Ok sir

**Comment by TomLucasakaTGeek** [comment]
Sir should I remove intro.ejs file and instead add the buttons at the top-right corner for Login or Signup as its more time-saving, or should I follow the exact scheme.

**Comment by Akshat111111** [comment]
yeah, do it
